<!DOCTYPE html>

<html lang="en">
<head>
  <title>Biomedical - Docs - Data Commons</title>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <link rel="icon" href="/assets/images/favicon.png" type="image/png"/>
  <link href="https://fonts.googleapis.com/css2?family=Google+Sans:wght@300;400;500;700" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css2?family=Google+Sans+Text:wght@300;400;500;700" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@600&family=Roboto&family=Material+Icons&display=swap" rel="stylesheet">
  <link href="/assets/css/styles.css" rel="stylesheet">
  <link rel="stylesheet" href="/assets/css/custom.css">
  
  
  <script src="https://public.flourish.studio/resources/embed.js"></script>
  
</head>
<body>
  <div id="main">
    <header id="main-header">
      <nav class="navbar navbar-light navbar-expand-lg col" id="main-nav">
        <div class="container-fluid">
          <div class="navbar-brand">
            <div id="main-header-logo">
                <a href="https://datacommons.org/">
                  <img src="/assets/images/dc-logo.svg" />
                </a>
            </div>
            <a href="https://datacommons.org">Data Commons</a>
            <span><a href="/">Docs</a></span>
            </a>
          </div>
        <button class="navbar-toggler" type="button" data-toggle="collapse"
          data-target="#dc-main-nav">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse justify-content-md-end" id="dc-main-nav">
          <ul class="navbar-nav float-md-right">
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" id="nav-explore-dropdown"
                role="button" data-toggle="dropdown" aria-haspopup="true"
                aria-expanded="false">
                Explore
              </a>
              <div class="dropdown-menu" aria-labelledby="nav-explore-dropdown">
                <a class="dropdown-item" href="https://datacommons.org/place">Place
                  Explorer</a>
                <a class="dropdown-item"
                  href="https://datacommons.org/browser">Knowledge Graph</a>
                <a class="dropdown-item"
                  href="https://datacommons.org/tools/visualization#visType=timeline">Timelines Explorer</a>
                <a class="dropdown-item"
                  href="https://datacommons.org/tools/visualization#visType=scatter">Scatter Plot Explorer</a>
                <a class="dropdown-item"
                  href="https://datacommons.org/tools/visualization#visType=map">Map Explorer</a>
                <a class="dropdown-item"
                  href="https://datacommons.org/tools/statvar">Statistical Variable Explorer</a>
                <a class="dropdown-item"
                  href="https://datacommons.org/tools/download">Data Download Tool</a>
              </div>
            </li>
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" id="nav-doc-dropdown"
                role="button" data-toggle="dropdown" aria-haspopup="true"
                aria-expanded="false">
                Documentation
              </a>
              <div class="dropdown-menu" aria-labelledby="nav-doc-dropdown">
                <a class="dropdown-item" href="/">Documentation</a>
                <a class="dropdown-item" href="/api">APIs</a>
                <a class="dropdown-item" href="/tutorials">Tutorials</a>
                <a class="dropdown-item" href="/contributing/">Contribute</a>
              </div>
            </li>
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" id="nav-about-dropdown"
                role="button" data-toggle="dropdown" aria-haspopup="true"
                aria-expanded="false">
                About
              </a>
              <div class="dropdown-menu" aria-labelledby="nav-about-dropdown">
                <a class="dropdown-item" href="https://datacommons.org/about">About Data
                  Commons</a>
                <a class="dropdown-item" href="https://blog.datacommons.org/">Blog</a>
                <a class="dropdown-item" href="https://datacommons.org/datasets">Data
                  Sources</a>
                <a class="dropdown-item" href="https://datacommons.org/faq">FAQ</a>
                <a class="dropdown-item" href="https://datacommons.org/feedback">Feedback</a>
              </div>
            </li>
            </ul>
            <div class="gcse-searchbox-only"
              data-resultsUrl="https://datacommons.org/search"></div>
            </div>
            </div>
      </nav>
    </header>

    <main id="" class="container-fluid">
      <div class="row">
        <div id="main-side-nav" class="col-3">
          <div id="sidebar"><details class="parent" open>
          <summary>
            <a href="/" class="">What is Data Commons?</a>
            
          </summary>
          </details><details class="parent" open>
          <summary>
            <a href="/how_to_use.html" class="">How to use Data Commons</a>
            
          </summary>
          </details><details class="parent" open>
          <summary>
            <a href="/data_model.html" class="">Key concepts</a>
            
          </summary>
          </details><details class="parent" open>
          <summary>
            <a href="/api/" class="">API</a>
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
          </summary>
          <ul id="node-96"><details class="child" >
                  <summary>
                    <a href="/api/rest/v2" class="">REST (v2)</a>
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
                  </summary>
                  <ul id="child-1"><details class="child" >
                          <summary>
                            <a href="/api/rest/v2/getting_started" class="">
                              
                              Getting Started Guide
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/rest/v2/observation" class="">
                              
                              Observation
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/rest/v2/node.html" class="">
                              
                              Node
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/rest/v2/resolve.html" class="">
                              
                              Resolve entities
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/rest/v2/sparql.html" class="">
                              
                              SPARQL
                            </a>
                            
                          </summary>
                          </details></ul></details><details class="child" >
                  <summary>
                    <a href="/api/python/" class="">Python</a>
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
                  </summary>
                  <ul id="child-4"><details class="child" >
                          <summary>
                            <a href="/api/python/query.html" class="">
                              
                              SPARQL
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/python/triple.html" class="">
                              
                              Triple
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/python/property_label.html" class="">
                              
                              Property Label
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/python/property_value.html" class="">
                              
                              Property Value
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/python/place_in.html" class="">
                              
                              Places within a Place
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/python/stat_value.html" class="">
                              
                              Place Statistics - Single Value
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/python/stat_series.html" class="">
                              
                              Place Statistics - Time Series
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/python/stat_all.html" class="">
                              
                              Place Statistics - All
                            </a>
                            
                          </summary>
                          </details></ul></details><details class="child" >
                  <summary>
                    <a href="/api/pandas/" class="">Pandas</a>
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
                  </summary>
                  <ul id="child-5"><details class="child" >
                          <summary>
                            <a href="/api/pandas/time_series.html" class="">
                              
                              Time Series as pd.Series
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/pandas/time_series_dataframe.html" class="">
                              
                              Time Series Table as pd.DataFrame
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/pandas/multivariate_dataframe.html" class="">
                              
                              Multivariate Table as pd.DataFrame
                            </a>
                            
                          </summary>
                          </details></ul></details><details class="child" >
                  <summary>
                    <a href="/api/sheets/" class="">Google Sheets</a>
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
                  </summary>
                  <ul id="child-6"><details class="child" >
                          <summary>
                            <a href="/api/sheets/get_name.html" class="">
                              
                              Names Associated with a DCID
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/sheets/places_in.html" class="">
                              
                              Places Contained Within Another Place
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/sheets/get_variable.html" class="">
                              
                              Statistical Variable Values
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/sheets/get_property.html" class="">
                              
                              Node Property Values
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/sheets/get_cohort_members.html" class="">
                              
                              Cohort Members
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/sheets/tutorials/" class="">
                              
                              Tutorials
                            </a>
                            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
                          </summary>
                          <ul id="grandchild-6"></ul></details></ul></details><details class="child" >
                  <summary>
                    <a href="/api/web_components/" class="">Web Components</a>
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
                  </summary>
                  <ul id="child-7"><details class="child" >
                          <summary>
                            <a href="/api/web_components/bar" class="">
                              
                              Bar Chart
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/web_components/gauge" class="">
                              
                              Gauge Chart
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/web_components/highlight" class="">
                              
                              Highlight Tile
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/web_components/line" class="">
                              
                              Line Chart
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/web_components/map" class="">
                              
                              Map Chart
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/web_components/pie" class="">
                              
                              Pie Chart
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/web_components/ranking" class="">
                              
                              Ranking Chart
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/web_components/scatter" class="">
                              
                              Scatter Plot
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/api/web_components/slider" class="">
                              
                              Slider Control
                            </a>
                            
                          </summary>
                          </details></ul></details></ul></details><details class="parent" open>
          <summary>
            <a href="/bigquery/" class="">Query with SQL/BigQuery</a>
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
          </summary>
          <ul id="node-97"><details class="child" >
                  <summary>
                    <a href="/bigquery/query_places.html" class="">Places</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/bigquery/query_property_places.html" class="">Properties of places</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/bigquery/query_more_complex.html" class="">More complex queries</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/bigquery/query_join_your_data.html" class="">Join with external data</a>
                    
                  </summary>
                  </details></ul></details><details class="parent" open>
          <summary>
            <a href="/papers/" class="">Papers</a>
            
          </summary>
          </details><details class="parent" open>
          <summary>
            <a href="/tutorials/" class="">Case Studies</a>
            
          </summary>
          </details><details class="parent" open>
          <summary>
            <a href="/courseware/" class="">Courseware</a>
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
          </summary>
          <ul id="node-100"><details class="child" >
                  <summary>
                    <a href="/courseware/data_literacy/" class="">Data Literacy with Data Commons</a>
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
                  </summary>
                  <ul id="child-1"><details class="child" >
                          <summary>
                            <a href="/courseware/data_literacy/overview.html" class="">
                              
                              Overview
                            </a>
                            
                          </summary>
                          </details><details class="child" >
                          <summary>
                            <a href="/courseware/data_literacy/course_materials/" class="">
                              
                              Course Materials
                            </a>
                            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
                          </summary>
                          <ul id="grandchild-2"><li class="">
                                  <a href="/courseware/data_literacy/course_materials/modules.html" class="">
                                    Modules
                                  </a>
                                </li><li class="">
                                  <a href="/courseware/data_literacy/course_materials/key_themes.html" class="">
                                    Key Themes
                                  </a>
                                </li><li class="">
                                  <a href="/courseware/data_literacy/course_materials/how_to_use.html" class="">
                                    How to Use
                                  </a>
                                </li></ul></details><details class="child" >
                          <summary>
                            <a href="/courseware/data_literacy/faqs.html" class="">
                              
                              FAQs
                            </a>
                            
                          </summary>
                          </details></ul></details><details class="child" >
                  <summary>
                    <a href="/courseware/intro_data_science.html" class="">Intro to Data Science</a>
                    
                  </summary>
                  </details></ul></details><details class="parent" open>
          <summary>
            <a href="/contributing/" class="">Contribute to Data Commons</a>
            
          </summary>
          </details><details class="parent" open>
          <summary>
            <a href="/custom_dc/" class="">Build your own Data Commons</a>
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
          </summary>
          <ul id="node-102"><details class="child" >
                  <summary>
                    <a href="/custom_dc/quickstart.html" class="">Get started</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/custom_dc/custom_data.html" class="">Work with custom data</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/custom_dc/custom_ui.html" class="">Customize the site</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/custom_dc/build_image.html" class="">Build and run a custom image</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/custom_dc/data_cloud.html" class="">Test data in Google Cloud</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/custom_dc/deploy_cloud.html" class="">Deploy a custom instance to Google Cloud</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/custom_dc/launch_cloud.html" class="">Launch a custom site</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/custom_dc/faq.html" class="">Frequently asked questions</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/custom_dc/troubleshooting.html" class="">Troubleshooting</a>
                    
                  </summary>
                  </details></ul></details><details class="parent" open>
          <summary>
            <a href="/place_types.html" class="">Place Types</a>
            
          </summary>
          </details><details class="parent" open>
          <summary>
            <a href="/datasets/" class="">Data Sources</a>
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
          </summary>
          <ul id="node-115"><details class="child" >
                  <summary>
                    <a href="/datasets/Agriculture.html" class="">Agriculture</a>
                    
                  </summary>
                  </details><details class="child" open>
                  <summary>
                    <a href="/datasets/Biomedical.html" class=" active">Biomedical</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/datasets/Crime.html" class="">Crime</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/datasets/Demographics.html" class="">Demographics</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/datasets/Economy.html" class="">Economy</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/datasets/Education.html" class="">Education</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/datasets/Energy.html" class="">Energy</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/datasets/Environment.html" class="">Environment</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/datasets/Health.html" class="">Health</a>
                    
                  </summary>
                  </details><details class="child" >
                  <summary>
                    <a href="/datasets/Housing.html" class="">Housing</a>
                    
                  </summary>
                  </details></ul></details><details class="parent" open>
          <summary>
            <a href="/import_dataset/" class="">Data Imports</a>
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="14" height="14">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.7803 6.21967C13.0732 6.51256 13.0732 6.98744 12.7803 7.28033L8.53033 11.5303C8.23744 11.8232 7.76256 11.8232 7.46967 11.5303L3.21967 7.28033C2.92678 6.98744 2.92678 6.51256 3.21967 6.21967C3.51256 5.92678 3.98744 5.92678 4.28033 6.21967L8 9.93934L11.7197 6.21967C12.0126 5.92678 12.4874 5.92678 12.7803 6.21967Z"></path>
</svg>
          </summary>
          <ul id="node-120"><details class="child" >
                  <summary>
                    <a href="/import_dataset/schema_less_guide.html" class="">Data Commons Schema-less Guide</a>
                    
                  </summary>
                  </details></ul></details><details class="parent" open>
          <summary>
            <a href="/glossary.html" class="">Glossary</a>
            
          </summary>
          </details><details class="parent" open>
          <summary>
            <a href="/blog.html" class="">Blog</a>
            
          </summary>
          </details></div>
        </div>
        <div class="col-9">
          <div id="main-doc-content">
            
              
                <nav aria-label="Breadcrumb" id="main-breadcrumb">
                  <ol class="breadcrumb">
                    
                      <li class="breadcrumb-item">
                        <a href="/datasets/">
                          Data Sources
                        </a>
                      </li>
                    
                    <li class="breadcrumb-item active"><span>Biomedical </span></li>
                  </ol>
                </nav>
              
            
            <h1 class="no_toc" id="biomedical">Biomedical</h1>

<ul id="markdown-toc">
  <li><a href="#broad-institute" id="markdown-toc-broad-institute">Broad Institute</a>    <ul>
      <li><a href="#gtex-analysis-v8-eqtl" id="markdown-toc-gtex-analysis-v8-eqtl">GTEx Analysis V8 eQTL</a></li>
    </ul>
  </li>
  <li><a href="#elixir-core-data-resources" id="markdown-toc-elixir-core-data-resources">ELIXIR Core Data Resources</a>    <ul>
      <li><a href="#the-molecular-interaction-mint-database" id="markdown-toc-the-molecular-interaction-mint-database">The Molecular INTeraction (MINT) Database</a></li>
    </ul>
  </li>
  <li><a href="#encyclopedia-of-dna-elements-encode" id="markdown-toc-encyclopedia-of-dna-elements-encode">Encyclopedia of DNA Elements (ENCODE)</a>    <ul>
      <li><a href="#experimental-data" id="markdown-toc-experimental-data">Experimental Data</a></li>
    </ul>
  </li>
  <li><a href="#european-molecular-biology-laboratory---european-bioinformatics-institute-embl-ebi" id="markdown-toc-european-molecular-biology-laboratory---european-bioinformatics-institute-embl-ebi">European Molecular Biology Laboratory - European Bioinformatics Institute (EMBL-EBI)</a>    <ul>
      <li><a href="#chembl" id="markdown-toc-chembl">ChEMBL</a></li>
      <li><a href="#uniprot" id="markdown-toc-uniprot">UniProt</a></li>
    </ul>
  </li>
  <li><a href="#gene-ontology-consortium" id="markdown-toc-gene-ontology-consortium">Gene Ontology Consortium</a>    <ul>
      <li><a href="#the-sequence-ontology" id="markdown-toc-the-sequence-ontology">The Sequence Ontology</a></li>
    </ul>
  </li>
  <li><a href="#international-committee-on-taxonomy-of-viruses-ictv" id="markdown-toc-international-committee-on-taxonomy-of-viruses-ictv">International Committee on Taxonomy of Viruses (ICTV)</a>    <ul>
      <li><a href="#master-species-list" id="markdown-toc-master-species-list">Master Species List</a></li>
      <li><a href="#virus-metadata-resource" id="markdown-toc-virus-metadata-resource">Virus Metadata Resource</a></li>
    </ul>
  </li>
  <li><a href="#jensen-lab-university-of-copenhagen" id="markdown-toc-jensen-lab-university-of-copenhagen">Jensen Lab (University of Copenhagen)</a>    <ul>
      <li><a href="#diseases" id="markdown-toc-diseases">DISEASES</a></li>
      <li><a href="#side-effect-resource-sider-41" id="markdown-toc-side-effect-resource-sider-41">Side Effect Resource (SIDER) 4.1</a></li>
    </ul>
  </li>
  <li><a href="#new-york-botanical-garden-nybg" id="markdown-toc-new-york-botanical-garden-nybg">New York Botanical Garden (NYBG)</a>    <ul>
      <li><a href="#c-v-starr-virtual-herbarium-collaboration" id="markdown-toc-c-v-starr-virtual-herbarium-collaboration">C. V. Starr Virtual Herbarium (Collaboration)</a></li>
    </ul>
  </li>
  <li><a href="#pharmgkb" id="markdown-toc-pharmgkb">PharmGKB</a>    <ul>
      <li><a href="#pharmgkb-primary-data" id="markdown-toc-pharmgkb-primary-data">PharmGKB Primary Data</a></li>
      <li><a href="#pharmgkb-relationships-data" id="markdown-toc-pharmgkb-relationships-data">PharmGKB Relationships Data</a></li>
    </ul>
  </li>
  <li><a href="#temporary-data-commons-data" id="markdown-toc-temporary-data-commons-data">Temporary Data Commons Data</a>    <ul>
      <li><a href="#temporary-gene-mappings" id="markdown-toc-temporary-gene-mappings">Temporary Gene Mappings</a></li>
    </ul>
  </li>
  <li><a href="#the-human-protein-atlas" id="markdown-toc-the-human-protein-atlas">The Human Protein Atlas</a>    <ul>
      <li><a href="#the-tissue-atlas" id="markdown-toc-the-tissue-atlas">The Tissue Atlas</a></li>
    </ul>
  </li>
  <li><a href="#us-adopted-names-usan-council" id="markdown-toc-us-adopted-names-usan-council">U.S. Adopted Names (USAN) Council</a>    <ul>
      <li><a href="#usan-stems" id="markdown-toc-usan-stems">USAN Stems</a></li>
    </ul>
  </li>
  <li><a href="#us-food-and-drug-administration-fda" id="markdown-toc-us-food-and-drug-administration-fda">U.S. Food and Drug Administration (FDA)</a>    <ul>
      <li><a href="#fda-approved-drugs" id="markdown-toc-fda-approved-drugs">FDA-Approved Drugs</a></li>
    </ul>
  </li>
  <li><a href="#us-national-institutes-of-health-national-center-for-biotechnology-information-nih-ncbi" id="markdown-toc-us-national-institutes-of-health-national-center-for-biotechnology-information-nih-ncbi">U.S. National Institutes of Health: National Center for Biotechnology Information (NIH: NCBI)</a>    <ul>
      <li><a href="#clinvar" id="markdown-toc-clinvar">ClinVar</a></li>
      <li><a href="#gene" id="markdown-toc-gene">Gene</a></li>
      <li><a href="#ncbi-taxonomy" id="markdown-toc-ncbi-taxonomy">NCBI Taxonomy</a></li>
    </ul>
  </li>
  <li><a href="#us-national-institutes-of-health-national-library-of-medicine" id="markdown-toc-us-national-institutes-of-health-national-library-of-medicine">U.S. National Institutes of Health: National Library of Medicine</a>    <ul>
      <li><a href="#medical-subject-headings-mesh" id="markdown-toc-medical-subject-headings-mesh">Medical Subject Headings (MeSH)</a></li>
      <li><a href="#pubchem" id="markdown-toc-pubchem">PubChem</a></li>
    </ul>
  </li>
  <li><a href="#ucsc-genomics-institute" id="markdown-toc-ucsc-genomics-institute">UCSC Genomics Institute</a>    <ul>
      <li><a href="#genome-browser" id="markdown-toc-genome-browser">Genome Browser</a></li>
    </ul>
  </li>
  <li><a href="#university-of-maryland-school-of-medicine-institute-of-genome-sciences" id="markdown-toc-university-of-maryland-school-of-medicine-institute-of-genome-sciences">University of Maryland School of Medicine, Institute of Genome Sciences</a>    <ul>
      <li><a href="#disease-ontology" id="markdown-toc-disease-ontology">Disease Ontology</a></li>
    </ul>
  </li>
  <li><a href="#world-health-organization-who" id="markdown-toc-world-health-organization-who">World Health Organization (WHO)</a>    <ul>
      <li><a href="#atc-codes" id="markdown-toc-atc-codes">ATC Codes</a></li>
    </ul>
  </li>
</ul>

<h3 id="broad-institute"><a href="https://www.broadinstitute.org/resources-services-and-tools">Broad Institute</a></h3>

<h4 id="gtex-analysis-v8-eqtl"><a href="https://www.gtexportal.org/home/datasets">GTEx Analysis V8 eQTL</a></h4>
<p>The GTEx eGene and significant variant-gene association data were generated from samples “collected from 54 non-diseased tissue sites across nearly 1000 individuals, primarily for molecular assays including WGS, WES, and RNA-Seq. Remaining samples are available from the GTEx Biobank.” The single-tissue cis-eQTL data from the v8 release was used.</p>

<p>GTEx is an NIH human genomic data unrestricted-access data repository and the data was made available in compliance with <a href="https://www.gtexportal.org/home/documentationPage#staticTextPublicationPolicy">GTEx Data Release and Publication Policy</a>. GTEx outlines <a href="https://www.gtexportal.org/home/faq#citePortal">how to cite</a> use of GTEx data in journal publication.</p>

<h3 id="elixir-core-data-resources"><a href="https://elixir-europe.org/platforms/data/core-data-resources">ELIXIR Core Data Resources</a></h3>

<h4 id="the-molecular-interaction-mint-database"><a href="https://mint.bio.uniroma2.it/">The Molecular INTeraction (MINT) Database</a></h4>
<p>The MINT Database “focuses on experimentally verified protein-protein interactions mined from the scientific literature by expert curators.”</p>

<p>Licata, Luana, Leonardo Briganti, Daniele Peluso, Livia Perfetto, Marta Iannuccelli, Eugenia Galeota, Francesca Sacco et al. “MINT, the molecular interaction database: 2012 update.” Nucleic acids research 40, no. D1 (2012): D857-D861.</p>

<h3 id="encyclopedia-of-dna-elements-encode"><a href="https://www.encodeproject.org/">Encyclopedia of DNA Elements (ENCODE)</a></h3>

<h4 id="experimental-data"><a href="https://www.encodeproject.org/help/project-overview/">Experimental Data</a></h4>
<p>The ENCODE experimental dataset contains information for approximately 7000 experiments along with 14,000 BED files collected by The Encyclopedia of DNA Elements (ENCODE) Consortium. Examples of experiment metadata captured include the target biosample, assay type, gene assembly, etc. Data Commons include the meta data for all experimental datasets in ENCODE as of 2019.</p>

<p>Data made available under: <a href="https://www.encodeproject.org/help/citing-encode/">ENCODE Data Use Policy for External Users</a>. This data was formatted for Data Commons through a collaboration with Dr. Anthony Oro’s group at Stanford University.</p>

<h3 id="european-molecular-biology-laboratory---european-bioinformatics-institute-embl-ebi"><a href="https://www.ebi.ac.uk/">European Molecular Biology Laboratory - European Bioinformatics Institute (EMBL-EBI)</a></h3>

<h4 id="chembl"><a href="https://www.ebi.ac.uk/chembl/">ChEMBL</a></h4>
<p>“ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs.” It includes information on drugs at all stages of drug discovery.</p>

<h4 id="uniprot"><a href="https://www.uniprot.org/">UniProt</a></h4>
<p>Data Commons includes protein sequence and functional information including protein interaction with chemical compounds maintained by the UniProt Consortium. The data is made available by the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution (CC BY 4.0) License</a>. Further information on UniProt License and Disclaimer can be found <a href="https://www.uniprot.org/help/license">here</a>. The UniProt Consortium states <a href="https://www.uniprot.org/help/publications">how to cite</a> UniProt data used in a journal article.</p>

<p>This data is made available by <a href="https://www.ebi.ac.uk/about/terms-of-use/">EMBL-EPI Terms of Use</a>.</p>

<h3 id="gene-ontology-consortium"><a href="https://geneontology.org/">Gene Ontology Consortium</a></h3>

<h4 id="the-sequence-ontology"><a href="http://www.sequenceontology.org/">The Sequence Ontology</a></h4>
<p>“The Sequence Ontology is a set of terms and relationships used to describe the features and attributes of biological sequence. SO includes different kinds of features which can be located on the sequence. Biological features are those which are defined by their disposition to be involved in a biological process. Examples are ‘binding site’ and ‘exon’. Biomaterial features are those which are intended for use in an experiment such as aptamer and PCR_product. There are also experimental features which are the result of an experiment. SO also provides a rich set of attributes to describe these features such as ‘polycistronic’ and ‘maternally imprinted’.”
Gene Ontology Consortium data and data products are licensed under the <a href="https://creativecommons.org/licenses/by/4.0/legalcode">Creative Commons Attribution 4.0 Unported License</a>. When using or citing GO data please mention the particular release. For example, include where applicable the date (e.g. ‘2024-01-17’), Zenodo DOI (e.g. ‘10.5281/zenodo.10536401’), and links. More information on licensing and attribution in regards to the Gene Ontology Consortium can be found <a href="https://geneontology.org/docs/go-citation-policy/">here</a>.</p>

<h3 id="international-committee-on-taxonomy-of-viruses-ictv"><a href="https://ictv.global/">International Committee on Taxonomy of Viruses (ICTV)</a></h3>

<h4 id="master-species-list"><a href="https://ictv.global/msl">Master Species List</a></h4>
<p>The official, current virus taxonomy approved by the ICTV. To accomplish the task of organizing and maintaining this virus taxonomy, the ICTV is composed of 7 subcommittees covering Animal DNA viruses and Retroviruses, Animal dsRNA and ssRNA (-) viruses, Animal ssRNA (+) viruses, Bacterial viruses, Archaeal Viruses, Fungal and Protist viruses, and Plant viruses. The ICTV has established over 100 international Study Groups (SGs) covering all major virus families and genera. The MSL version currently in the graph is MSL38 v3 released on 2023-09-11.</p>

<h4 id="virus-metadata-resource"><a href="https://ictv.global/vmr">Virus Metadata Resource</a></h4>
<p>The ICTV chooses an exemplar virus for each species and the VMR provides a list of these exemplars. An exemplar virus serves as an example of a well-characterized virus isolate of that species and includes the GenBank accession number for the genomic sequence of the isolate as well as the virus name, isolate designation, suggested abbreviation, genome composition, and host source. The VMR version currently in the graph is VMR MSL38 v2 released on 2023-09-13.
This data is made available under Creative Commons Attribution ShareAlike 4.0 International (CC BY-SA 4.0).</p>

<h3 id="jensen-lab-university-of-copenhagen"><a href="https://jensenlab.org/resources/">Jensen Lab (University of Copenhagen)</a></h3>

<h4 id="diseases"><a href="https://diseases.jensenlab.org/Search">DISEASES</a></h4>
<p>DISEASES is a weekly updated web resource that integrates evidence on disease-gene associations from automatic text mining, manually curated literature, cancer mutation data, and genome-wide association studies. This dataset further unifies the evidence by assigning confidence scores that facilitate comparison of the different types and sources of evidence. For further details please refer to the following Open Access articles about the database: <a href="https://www.sciencedirect.com/science/article/pii/S1046202314003831">DISEASES: Text mining and data integration of disease-gene associations</a> and <a href="https://academic.oup.com/database/article/doi/10.1093/database/baac019/6554833?login=false">DISEASES 2.0: a weekly updated database of disease–gene associations from text mining and data integration</a>. The data is made available under the <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY</a> license.</p>

<h4 id="side-effect-resource-sider-41"><a href="http://sideeffects.embl.de/">Side Effect Resource (SIDER) 4.1</a></h4>
<p>SIDER is a database of adverse drug reactions. Available information includes side effect frequency, drug and side effect classifications as well as links to further information, for example drug–target relations. However, this database uses MEDRA ontology, which is under the UMLS license that is limited to non-commercial use. Therefore, only the data under zero license - mappings of PubChem Compound IDs (CIDs), and ATC Codes - are hosted. Data Commons hosts version 4.1 of SIDER released on October 21, 2015. Information about citing SIDER can be found <a href="http://sideeffects.embl.de/about/">here</a>.</p>

<p>This data is made available under the <a href="https://creativecommons.org/publicdomain/zero/1.0/">CC0 1.0 Universal (CC0 1.0) Public Domain Dedication</a>.</p>

<h3 id="new-york-botanical-garden-nybg"><a href="http://sweetgum.nybg.org/science/">New York Botanical Garden (NYBG)</a></h3>

<h4 id="c-v-starr-virtual-herbarium-collaboration"><a href="http://sweetgum.nybg.org/science/vh/learn-more/">C. V. Starr Virtual Herbarium (Collaboration)</a></h4>
<p>C. V. Starr Virtual Herbarium is a public specimen database with photos and detailed records about millions of plants, fungi, and algae.</p>

<h3 id="pharmgkb"><a href="https://www.pharmgkb.org/">PharmGKB</a></h3>

<h4 id="pharmgkb-primary-data"><a href="https://www.pharmgkb.org/">PharmGKB Primary Data</a></h4>
<p>The Pharmacogenomics Knowledge Base, PharmGKB, is an interactive tool for researchers investigating how genetic variation affects drug response. The PharmGKB Web site, http://www.pharmgkb.org, displays genotype, molecular, and clinical knowledge integrated into pathway representations and Very Important Pharmacogene (VIP) summaries with links to additional external resources. Users can search and browse the knowledgebase by genes, variants, drugs, diseases, and pathways. The Primary Data contains summary information on chemicals, drugs, genes, genetic variants, and phenotypes.</p>

<h4 id="pharmgkb-relationships-data"><a href="https://www.pharmgkb.org/">PharmGKB Relationships Data</a></h4>
<p>PharmGKB reports association between chemicals, diseases, genes, and genetic variants, both with themselves and with each other.</p>

<p>Data made available under Creative Commons Attribution-ShareAlike 4.0 Intergovernmental Organization (CC BY-SA 4.0 IGO) licence. Explicit licensing for PharmGKB can be viewed on the <a href="https://www.pharmgkb.org/downloads">download page</a>.</p>

<h3 id="temporary-data-commons-data"><a href="https://www.datacommons.org/">Temporary Data Commons Data</a></h3>

<h4 id="temporary-gene-mappings"><a href="https://www.datacommons.org/">Temporary Gene Mappings</a></h4>
<p>This maps the new way of generating Gene dcids (bio/<gene_symbol>) with the old, preexisting Gene dcids(bio/<genome_assembly>_<gene_symbol>). These are temporary mappings until all data using the old method of Gene dcid generation has been updated.</gene_symbol></genome_assembly></gene_symbol></p>

<p>Data is publicly available via Data Commons.</p>

<h3 id="the-human-protein-atlas"><a href="https://www.proteinatlas.org/">The Human Protein Atlas</a></h3>

<h4 id="the-tissue-atlas"><a href="https://www.proteinatlas.org/humanproteome/tissue">The Tissue Atlas</a></h4>
<p>The Human Protein Tissue Atlas contains information about the distribution of proteins on human tissues derived from the antibody-based protein profiling from 44 normal human tissues types and mRNA expression data from 37 different normal tissue types.</p>

<p>This <a href="https://www.proteinatlas.org/download/normal_tissue.tsv.zip">dataset</a> is available under <a href="https://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0</a>. Please also see their <a href="https://www.proteinatlas.org/about/disclaimer">Disclaimer</a> and <a href="https://www.proteinatlas.org/about/licence">Licence &amp; Citation</a>.</p>

<h3 id="us-adopted-names-usan-council"><a href="https://www.ama-assn.org/about/united-states-adopted-names/usan-council">U.S. Adopted Names (USAN) Council</a></h3>

<h4 id="usan-stems"><a href="https://www.ama-assn.org/about/united-states-adopted-names/united-states-adopted-names-approved-stems">USAN Stems</a></h4>
<p>USAN stems represent common stems for which chemical and/or pharmacologic parameters have been established. These council-approved stems and their definitions are recommended for use in coining new nonproprietary drug names belonging to an established series of related agents. USAN appropriately incorporates this established class stem system. By doing so, similar compounds maintain a common “family” name that provides immediate recognition.</p>

<p>This data is made available through <a href="https://open.fda.gov/license/">openFDA terms of service</a>.</p>

<h3 id="us-food-and-drug-administration-fda"><a href="https://www.fda.gov/">U.S. Food and Drug Administration (FDA)</a></h3>

<h4 id="fda-approved-drugs"><a href="https://www.accessdata.fda.gov/scripts/cder/daf/">FDA-Approved Drugs</a></h4>
<p>“Drugs@FDA includes information about drugs, including biological products, approved for human use in the United States.” Data Commons includes the information about the FDA application for the drug as well as the drug’s strength, active ingredients, dosage forms, administration routes, FDA therapeutic equivalence code, and marketing status.</p>

<p>This data is made available through <a href="https://open.fda.gov/license/">openFDA terms of service</a>.</p>

<h3 id="us-national-institutes-of-health-national-center-for-biotechnology-information-nih-ncbi"><a href="https://www.ncbi.nlm.nih.gov/">U.S. National Institutes of Health: National Center for Biotechnology Information (NIH: NCBI)</a></h3>

<h4 id="clinvar"><a href="https://www.ncbi.nlm.nih.gov/clinvar/">ClinVar</a></h4>
<p>“ClinVar is a freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence.” It contains reports of genetic “variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data.” Data Commons includes the January 6, 2020 release of the ClinVar archive supporting both hg19 and hg38 genome assemblies.</p>

<h4 id="gene"><a href="https://www.ncbi.nlm.nih.gov/gene">Gene</a></h4>
<p>The NIH NCBI gene info datasets from NCBI Gene for a subset of species contains “gene-specific content based on NCBI’s RefSeq project, information from model organism databases, and links to other resources.” The NCBI RefSeq project is “a comprehensive, integrated, non-redundant, well-annotated set of reference sequences including genomic, transcript, and protein”. The datasets included are from the February 19, 2020 update. The gene info files for the following species have been added:</p>

<ul>
  <li><em>Caenorhabditis elegans</em></li>
  <li><em>Danio rerio</em></li>
  <li><em>Drosophila melanogaster</em></li>
  <li><em>Gallus gallus</em></li>
  <li><em>Homo sapiens</em></li>
  <li><em>Mus musculus</em></li>
  <li><em>Saccharomyces cerevisiae</em></li>
  <li><em>Xenepus laevis</em></li>
</ul>

<h4 id="ncbi-taxonomy"><a href="https://www.ncbi.nlm.nih.gov/taxonomy/">NCBI Taxonomy</a></h4>
<p>“NCBI Taxonomy “consists of a curated set of names and classifications for all of the source organisms represented in the International Nucleotide Sequence Database Collaboration (INSDC). The NCBI Taxonomy database contains a list of names that are determined to be nomenclaturally correct or valid (as defined according to the different codes of nomenclature), classified in an approximately phylogenetic hierarchy (depending on the level of knowledge regarding phylogenetic relationships of a given group) as well as a number of names that exist outside the jurisdiction of the codes. That is, it focuses on nomenclature and systematics, rather than documenting the description of taxa.”</p>

<p>This data is from an NIH human genome unrestricted-access data repository and made accessible under the <a href="https://osp.od.nih.gov/scientific-sharing/genomic-data-sharing/">NIH Genomic Data Sharing (GDS) Policy</a>.</p>

<h3 id="us-national-institutes-of-health-national-library-of-medicine"><a href="https://www.nlm.nih.gov/">U.S. National Institutes of Health: National Library of Medicine</a></h3>

<h4 id="medical-subject-headings-mesh"><a href="https://meshb.nlm.nih.gov/search">Medical Subject Headings (MeSH)</a></h4>
<p>The Medical Subject Headings (MeSH) thesaurus is a controlled and hierarchically-organized vocabulary produced by the National Library of Medicine. It is used for indexing, cataloging, and searching of biomedical and health-related information. Data Commons includes the Concept, Descriptor, Qualifier, Supplementary Concept Record, and Term elements of MeSH as described <a href="https://www.nlm.nih.gov/mesh/xml_data_elements.html">here</a> defined by all four xml files provided by MeSH (desc, pa, qual, and supp). Data Commons includes production year 2024 MeSH.</p>

<h4 id="pubchem"><a href="https://pubchem.ncbi.nlm.nih.gov/">PubChem</a></h4>
<p>PubChem is the world’s largest collection of freely accessible chemical information. Search chemicals by name, molecular formula, structure, and other identifiers. Find chemical and physical properties, biological activities, safety and toxicity information, patents, literature citations and more.</p>

<p>This data is from the National Library of Medicine (NLM) and is not subject to copyright and is freely reproducible as stated in the <a href="https://www.nlm.nih.gov/web_policies.html">NLM’s copyright policy</a>.</p>

<h3 id="ucsc-genomics-institute"><a href="https://genome.ucsc.edu/">UCSC Genomics Institute</a></h3>

<h4 id="genome-browser"><a href="https://genome.ucsc.edu/">Genome Browser</a></h4>
<p>The UCSC Genome Browser originated from The Human Genome Project in 2000 to share and visualize genome data. It has grown to include an agglomeration of various genome assemblies and annotations. Data Commons includes data annotating chromosomes, genes, RNA transcripts, and genetic variants from the UCSC Genome Browser. The .chrom.sizes.txt files were downloaded from the UCSC Genome Browser Downloads page on August 13, 2019. The NCBI RefSeq files were downloaded from the UCSC Table Browser on August 2, 2019 for the following genome assemblies:</p>

<ul>
  <li>ce10</li>
  <li>ce11</li>
  <li>danRer10</li>
  <li>danRer11</li>
  <li>dm3</li>
  <li>dm6</li>
  <li>galGal5</li>
  <li>galGal6</li>
  <li>hg19</li>
  <li>hg38</li>
  <li>mm9</li>
  <li>mm10</li>
  <li>sacCer3</li>
  <li>xenLae2</li>
</ul>

<p>The All SNPs files were downloaded from the UCSC Table Browser on August 13, 2019 for the following genome assemblies and dbSNP builds:</p>

<ul>
  <li>gaGal5 (dbSNP Build 147)</li>
  <li>hg19 (dbSNP Build 151)</li>
  <li>hg38 (dbSNP Build 151)</li>
  <li>mm9 (dbSNP Build 128)</li>
  <li>mm10 (dbSNP Build 142)</li>
</ul>

<p>The annotation data is made freely available under the UCSC Genome Browser <a href="https://genome.ucsc.edu/conditions.html">terms of use</a>. The UCSC Genome Browser states <a href="https://genome.ucsc.edu/cite.html">how to cite</a> use of their data in a journal article publication.</p>

<h3 id="university-of-maryland-school-of-medicine-institute-of-genome-sciences"><a href="https://www.igs.umaryland.edu/">University of Maryland School of Medicine, Institute of Genome Sciences</a></h3>

<h4 id="disease-ontology"><a href="https://disease-ontology.org/">Disease Ontology</a></h4>
<p>The Disease Ontology was developed as a project by the Institute of Genome Sciences at the University of Maryland School of Medicine. It “is a community driven, open source ontology that is designed to link disparate datasets through disease concepts”. It provides a “standardized ontology for human disease with the purpose of providing the biomedical community with consistent, reusable and sustainable descriptions of human disease terms, phenotype characteristics and related medical vocabulary disease concepts”.</p>

<p>The data is made available under <a href="https://disease-ontology.org/resources/citing-do">C0 1.0 Universal (CC0 1.0) Public Domain Dedication</a>.</p>

<h3 id="world-health-organization-who"><a href="https://www.who.int/">World Health Organization (WHO)</a></h3>

<h4 id="atc-codes"><a href="https://www.whocc.no/atc_ddd_index/">ATC Codes</a></h4>
<p>Anatomical Therapeutic Chemical (ATC) is a heirarchical classification system for pharmacological substances. ‘In the ATC classification system, the active substances are classified in a hierarchy with five different levels. The system has fourteen main anatomical/pharmacological groups or 1st levels. Each ATC main group is divided into 2nd levels which could be either pharmacological or therapeutic groups. The 3rd and 4th levels are chemical, pharmacological or therapeutic subgroups and the 5th level is the chemical substance. The 2nd, 3rd and 4th levels are often used to identify pharmacological subgroups when that is considered more appropriate than therapeutic or chemical subgroups.’</p>

<p>Data made available under <a href="https://www.who.int/about/policies/publishing/copyright">CC BY-NC-SA 3.0 IGO</a>.</p>


          </div>
        </div>
      </div>
    </main>

    <footer id="main-footer">
      <div id="sub-footer">
        <span class="brand-byline">
          <span class="brand-text">An initiative from</span>
          <img class="brand-logo" src="/assets/images/google-logo.svg" />
        </span>
        <div class="sub-footer-links">
          <a href="https://policies.google.com/terms">Terms and Conditions</a>
          <a href="https://policies.google.com/privacy?hl=en-US">Privacy Policy</a>
          <a href="https://datacommons.org/disclaimers">Disclaimers</a>
        </div>
      </div>
    </footer>
  </div>
  <script
    src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script
    src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
    integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
    crossorigin="anonymous"></script>
  <script async
    src="https://cse.google.com/cse.js?cx=010079880385165835740%3Aosnv3q-sw08"></script>
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-KWSES5WXZE"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());
    gtag('config', 'G-KWSES5WXZE', {
      client_storage: 'none',
      anonymize_ip: true,
    });
  </script>
  <script src="/assets/js/tabs.js"></script>
</body>
</html>
